go.mod

```diff
@@ -87,9 +88,9 @@ require (
 	gopkg.in/square/go-jose.v2 v2.5.1
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776
-	k8s.io/api v0.20.2
+	k8s.io/api v0.21.1
```

Comment:
please lower version to original, by lowering it in github.com/wozniakjan/reverse-kube-resource